cellmlmanip/model.py

```diff
@@ -630,7 +630,7 @@ def graph_with_sympy_numbers(self):
             # And replace the equation with one with the rhs subbed with sympy.Number objects
             if subs_dict:
                 # Update rhs
-                rhs = equation.rhs.xreplace(subs_dict)
+                rhs = equation.rhs.subs(subs_dict)
```

Comment:
I thought `xreplace` was OK here?

Reply:
Or is `subs` somehow faster than `xreplace`?

Reply:
Would love some numbers to see what the difference in speed between the various options is :)

Reply:
@jonc125 right so here is what I did:
put the derivative eqs & symbols for the bondarenko_model_2004_apex model in a python script and perform a partial_eval style replacement (everything substituted into the derivative equations). 10 times for each method on my pc. The following is in time.process_time()
xreplace took:12.015625
subs took:126.5
replace (in a loop over subs_dict) took:29.234375
xreplace (in a loop over subs_dict) took:28.734375

Reply:
(changed it to xreplace)